example/android/gradle.properties

```diff
@@ -29,7 +29,7 @@ android.enableJetifier=true
 
 # Version of Flipper to use with React Native. Default value is whatever React
 # Native defaults to. To disable Flipper, set it to `false`.
```

Comment:
We should probably also change the comment now that Flipper is off by default.

Reply:
Also just realized that it's on by default on iOS as well, so we should either disable both or just leave it as is for now?

Reply:
Let's disable it on iOS as well:
https://github.com/microsoft/react-native-test-app/blob/d0292b20fed1d4afaa8382a85d65379f7fe3e1a2/ios/test_app.rb#L55-L61

Then do a minor bump. It's a backwards compatible change so it should be fine.